src/frameworks/core_graphics/cg_image.rs

```diff
@@ -120,11 +121,17 @@ fn CGImageGetHeight(env: &mut Environment, image: CGImageRef) -> GuestUSize {
     height
 }
 
+fn CGImageGetDataProvider(_env: &mut Environment, image: CGImageRef) -> CGDataProviderRef {
+    // TODO: implement proper provider?
```

Comment:
Can you add a comment mentioning the hack in cf_data.rs and cg_data_provider.rs?

Reply:
Done